### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RobertPlaiasu**

### CODE REVIEW

File: src/app/Policies/AdminPolicyTrait.php [hunk 1]
```diff
@@ -12,7 +12,8 @@ trait AdminPolicyTrait
 
     public function before(User $user)
     {
-        return $user->role_id == Role::where('name','administrator')->first()->id;
+        if($user->role_id == Role::where('name','administrator')->first()->id)
+            return true;
```
Comment (GrigoAleex) [inline src/app/Policies/AdminPolicyTrait.php:16]: Why did you change this?

**Review by GrigoAleex — APPROVED** [review]

File: src/app/Policies/AdminPolicyTrait.php (hunk 1 above)
Comment (RobertPlaiasu) [reply src/app/Policies/AdminPolicyTrait.php:16]: i did something wrong before needs to return null
